src/main/webapp/WEB-INF/spring-config.xml

```diff
@@ -154,7 +154,7 @@
 		<security:intercept-url pattern="/enterSignupOTP*"
 			access="permitAll" />
 		<security:intercept-url pattern="/**"
-			access="isAuthenticated()" />
+			access="permitAll" />

```

Comment:
Why you removed this..This is required for so not to allow unAuthenticated access.